release/ray_release/byod/byod_rllib.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -exo pipefail
+
+pip install gymnasium==1.1.1 ale_py==0.10.1 imageio==2.34.2 opencv-python-headless==4.9.0.80 wandb
+pip install torch==2.7 torchvision==0.22 --index-url https://download.pytorch.org/whl/cu128
```

Comment:
Dumb question: Is this defined for a specific machine type? Or could it be that the CUDA version plays a role for different GPU types?

Reply:
I guess the different cuda version in torch are made for different GPU types or machines, e.g. T4 has I think only CUDA 12.4